draft-ietf-scitt-architecture.md

```diff
@@ -331,12 +331,15 @@ Before an Issuer is able to produce Signed Statements, it must first create its
 A DID can be *resolved* into a *key manifest* (a list of public keys indexed by a *key identifier*) using many different DID methods.
 
 Issuers MAY choose the DID method they prefer, but with no guarantee that all Transparency Services will be able to register their Signed Statements.
```

Comment:
TS must advertise the DID methods they support.

Independently, implementations MUST support did:web (but they may be configured not to accept it)